### ISSUE TITLE
Exporting images corrupting file.

### ISSUE THREAD

**Issue by joeberth** [gitbrent/PptxGenJS#578]
Hello,
I try to export images, but the files are coming corrupted. I tried to use the fiddle from the PptxGenJs site, but it returns the same error. Does anyone have any solutions?

Below are links to images that corrupt the files.

https://external.xx.fbcdn.net/safe_image.php?d=AQASRShs-_UJVAtl&w=720&h=720&url=https%3A%2F%2Fwww.facebook.com%2Fads%2Fimage%2F%3Fd%3DAQLsD3JiAaNukB9m4zVUptSEB-OMClKdHDkDpeiHsKEq7CLSQVIfadsSN987EBMtRGyoJE_hg0ynPoWcXpg5z2jTo-VYjVHXzltIyCpqTC0zVcQxBMYvrLDumv2OkfjUCVY_UPBMi5H3IXWcZI8gJoVC&cfs=1&_nc_hash=AQABY3ez00WCYvnS

http://www.entremulherestv.com.br/wp-content/uploads/2019/06/CocaZero-RestauranteWeek.jpg

Thanks!

**Comment by gitbrent** [comment]
What does your code look like?

Have you looked at the sample image code under `/examples`?

### [Image API Docs](https://gitbrent.github.io/PptxGenJS/docs/api-images.html)

**Comment by joeberth** [comment]
Yeah, i coded the application looking looking at the api and the examples, almost everything worked perfectly, just a few inconsistencies I found, that's one of them.

https://jsfiddle.net/ypsojehg/4/

This fiddle is an example of a path that corrupts the file.
I have tried many times and received a few different errors, i am working in a api that receive path's from facebook like this one above and export in pptx. 

Another inconsistency is about this second fiddle, the image does not corrupt the file, but it opens as if the image were a file error.

https://jsfiddle.net/mstj3Lpq/

This is one of the main parts of my application, I appreciate any help and attention.

ps: Sorry for use the website fiddle to reproduce my error.

**Comment by fabiosl** [comment]
+1 here

**Comment by fabiosl** [comment]
Same thing here @gitbrent

**Comment by gregroyal** [comment]
Same thing here

**Comment by rusticpenguin** [comment]
I've ran into this when exporting a base64 image
